### DISCUSSION TITLE
Bulk results for a range of dates

### DISCUSSION THREAD

**Discussion by hidp123** [AminKH/MoonSun#2] [Q&A] [answered]
Hi, is it possible to generate results for example one year? So moon visibility data for all the months in a year. Or sunset times for say one year.

**Comment by AminKH** [comment]
Hi, Yes it is possible. These data are in calendar, for every day.

**Reply by hidp123** [reply]
Can I generate data, for example 5 years, in one go?

**Comment by AminKH** [comment, marked as answer]
No, sorry, it is for one year. But you can make for five consecutive years.
Output is text and copy and paste to larger file. I encountered  memory
problems for one year, let alone five years.
